lib/mwoffliner.lib.js

```diff
@@ -911,6 +927,99 @@ module.exports = {
                 }
             }
 
+            function storeDependencies( parsoidDoc, articleId, finished ) {
+                const filenameRadical = computeFilenameRadical()
+
+                // the encodeURI bellow is mandatory for languages with illegal letters for uri (like fa.wikipedia.org)
+                fetch(encodeURI(`${apiUrl}action=parse&format=json&page=${articleId}&prop=modules|jsconfigvars`), {
```

Comment:
Kelson, I talked to Skylsmoi and it's not possible to avoid this call : the call to api also returns js modules which are not included in the html deliverd in the example you give.

Removing this API call will result in missing modules.